### PR TITLE
PBFT proposal blocks on testnet

### PR DESCRIPTION
**Pull request by gaoqi8649**
As @justinsnapp require, PBFT proposal blocks rate change to depends on stakes, not every delegation node has the same rate anymore.
Since we rewards all players on top 200 for produced PBFT blocks on testnet. I am not sure if all players have the same staking. If not, the players will have a different proposal rate.

### CODE REVIEW

**Comment by JakubFornadel** [comment]
@gaoqi8649 can we close this PR ? Are we going to use it ?

**Comment by gaoqi8649** [comment]
> @gaoqi8649 can we close this PR ? Are we going to use it ?

As discussed, we will terminate testnet branch and instead of using develop brach testing on testnet. If like that, we should close the PR.

**Review by MatusKysel — APPROVED** [review]

**Comment by mfrankovi** [comment]
Based on previous comments, I believe this should not be merged to testnet. We will merge this functionality once we merge develop to testnet. Closing PR

**Pull request closed without merging**
